deploy/helm/kubecf/assets/operations/instance_groups/eirini.yaml

```diff
@@ -148,9 +148,9 @@
           certs_secret_name: eirini-staging-secret
           cc_internal_api: https://cloud-controller-ng.service.cf.internal:9023
           eirini_address: {{ printf "https://eirini.%s:8484" .Release.Namespace }}
-          downloader_image: "eirini/recipe-downloader:0.3.0"
-          uploader_image: "eirini/recipe-uploader:0.3.0"
-          executor_image: "eirini/recipe-executor:0.3.0"
+          downloader_image: "eirini/recipe-downloader@sha256:80c213363762d3171fe401c331c480f0e8eb34fe781798eda95de1e7427b2771"
+          uploader_image: "eirini/recipe-uploader@sha256:cb70f76035866bf280e710247751f2f620c4c6e6e6950a6e730bde01535bebf1"
+          executor_image: "eirini/recipe-executor@sha256:c072f828b86766778d027041e394eaad9cfb55c3b5f652a50be8455d1dad3bf4"
```

Comment:
We should have a follow-up task to make this customizable, like all the other releases are.

Reply:
created: https://github.com/cloudfoundry-incubator/kubecf/issues/640